vm/devices/storage/disk_backend_resources/src/lib.rs

```diff
@@ -8,37 +8,29 @@
 
 use mesh::MeshPayload;
 use vm_resource::kind::DiskHandleKind;
+use vm_resource::kind::DiskLayerHandleKind;
+use vm_resource::IntoResource;
 use vm_resource::Resource;
 use vm_resource::ResourceId;
 
 // Define config types here so that you don't have to pull in the individual
 // crates just to describe the configuration.
 
-/// RAM disk handle.
+/// RAM disk layer handle.
 ///
 /// FUTURE: allocate shared memory here so that the disk can be migrated between
 /// processes.
 #[derive(MeshPayload)]
-pub struct RamDiskHandle {
-    /// Size of the disk, in bytes.
-    pub len: u64,
+pub struct RamDiskLayerHandle {
```

Comment:
can you organize these layer handles into their own `mod layer`?

Reply:
Yeah OK